src/celengine/glsupport.cpp

```diff
@@ -0,0 +1,57 @@
+#include "glsupport.h"
+
+namespace celestia
+{
+namespace gl
+{
+
+bool ARB_shader_texture_lod         = false;
+bool ARB_vertex_array_object        = false;
+bool EXT_framebuffer_object         = false;
+bool EXT_texture_compression_s3tc   = false;
+bool EXT_texture_filter_anisotropic = false;
+
+namespace
```

Comment:
Why empty namespace?

Reply:
to make function local to the file. it's a c++ way instead of making functions `static`